### PR TITLE
Unified log tracing id of storage read request.

### LINKED ISSUE #6027: Log tracing id of storage read request.
## Enhancement

In `DeltaMergeStore::read`, there are two strings `tracing_id` and `req_info` play the role of the log tracing id. However, their contents have little differences:

- `tracing_id` contains `MPPTaskId` and `TableId`.
- `req_info` only contains `MPPTaskId`.

I think we can remove `req_info`.

image

### PR DESCRIPTION
**Pull request by JinheLin**
### What problem does this PR solve?

Issue Number: close #6027 

### Check List

Tests 

- [ ] Unit test
- [x] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

```release-note
None
```

### CODE REVIEW

File: dbms/src/Storages/DeltaMerge/DeltaMergeStore.cpp [hunk 1]
```diff
@@ -1776,5 +1773,20 @@ SegmentReadTasks DeltaMergeStore::getReadTasksByRanges(
     return tasks;
 }
 
+String DeltaMergeStore::getLogTracingId(const DMContext & dm_ctx)
+{
+    if (!dm_ctx.tracing_id.empty())
```
Comment (SeaRise) [inline dbms/src/Storages/DeltaMerge/DeltaMergeStore.cpp:1778]:
```suggestion
    if (likely(!dm_ctx.tracing_id.empty()))
```

**Review by SeaRise — APPROVED** [review]
LGTM

File: dbms/src/Storages/DeltaMerge/DeltaMergeStore.cpp [hunk 2]
```diff
@@ -1776,5 +1773,20 @@ SegmentReadTasks DeltaMergeStore::getReadTasksByRanges(
...
+    }
+    else if (const auto * dag_ctx = dm_ctx.db_context.getDAGContext(); dag_ctx != nullptr && dag_ctx->isMPPTask())
+    {
+        return fmt::format("{} Table<{}>", dag_ctx->getMPPTaskId().toString(), physical_table_id);
+    }
+    else
+    {
+        return fmt::format("Table<{}>", physical_table_id);
+    }
+}
```
Comment (JaySon-Huang) [inline dbms/src/Storages/DeltaMerge/DeltaMergeStore.cpp:1790]: the `tracing_id` is set here:
https://github.com/pingcap/tiflash/blob/2f215f57a12c819d73528534c08098028160f49a/dbms/src/Flash/Coprocessor/DAGStorageInterpreter.cpp#L660
https://github.com/pingcap/tiflash/blob/2f215f57a12c819d73528534c08098028160f49a/dbms/src/Flash/Coprocessor/DAGStorageInterpreter.cpp#L293

If we change the code in DAGStorageInterpreter, the string defined in L1782 and L1785 will become incorrect. Maybe simplify it as follow is OK? /cc @windtalker 
```suggestion
String DeltaMergeStore::getLogTracingId(const DMContext & dm_ctx)
{
    if (!dm_ctx.tracing_id.empty())
    {
        return dm_ctx.tracing_id;
    }
    else
    {
        return fmt::format("Table<{}>", physical_table_id);
    }
}
```

Comment (SeaRise) [reply dbms/src/Storages/DeltaMerge/DeltaMergeStore.cpp:1790]: lgtm

**Review by JaySon-Huang — APPROVED** [review]

**Pull request merged**